index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+<body>
+  <section class="section-header flex gen-padding">
+    <a href="#" class="logo-box">
+      <img class="logo" src="img/logo.png" alt="logo">
+    </a>
+    <a href="#" class="menu-box">
+      <img class="menu-icon" src="img/menu.png" alt="menu icon">
+    </a>
+  </section>
```

Comment:
- Hey, this should be a header tag as this holds the introductory content of your webpage such as the nav items. let's make this a header tag